ci-operator/config/open-cluster-management/search-collector/open-cluster-management-search-collector-main.yaml

```
@@ -0,0 +1,21 @@
+build_root:
+  image_stream_tag:
+    name: builder
+    namespace: open-cluster-management
+    tag: go1.16-linux-amd64
+images:
+- dockerfile_path: Dockerfile
+  from: base
+  to: search-collector
+promotion:
+  name: "2.3"
+  namespace: open-cluster-management
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
```

Requires this PR: https://github.com/open-cluster-management/search-collector/pull/108

```suggestion
      memory: 200Mi
      - artifact_dir: /tmp/artifacts
      as: unit
      commands: |
        export SELF="make -f Makefile.prow"
        make -f Makefile.prow unit-tests
      container:
        from: bin
```